packages/web3torrent/src/pages/file/File.tsx

```diff
@@ -46,41 +46,33 @@ const File: React.FC<Props> = props => {
     (torrent.status !== Status.Idle || !!torrent.originalSeed) && 1000
   );
 
+  const {channelCache, budgetCache, mySigningAddress: me} = web3Torrent.paymentChannelClient;
+  // TODO: We shouldn't have to check all these different conditions
+  const showBudget =
+    !!budgetCache &&
+    !_.isEmpty(budgetCache) &&
+    !!budgetCache.budgets &&
+    budgetCache.budgets.length > 0;
+
   return (
     <section className="section fill download">
       <div className="jumbotron-upload">
         <h1>{torrent.originalSeed ? 'Upload a File' : 'Download a File'}</h1>
       </div>
-      <Web3TorrentContext.Consumer>
-        {web3Torrent => {
-          const {
-            channelCache,
-            budgetCache,
-            mySigningAddress: me
-          } = web3Torrent.paymentChannelClient;
-          // TODO: We shouldn't have to check all these different conditions
-          const showBudget =
-            !!budgetCache &&
-            !_.isEmpty(budgetCache) &&
-            !!budgetCache.budgets &&
-            budgetCache.budgets.length > 0;
-          return (
-            <>
-              <TorrentInfo torrent={torrent} channelCache={channelCache} mySigningAddress={me} />
-              <br />
-              {showBudget ? (
-                <SiteBudgetTable
-                  budgetCache={budgetCache}
-                  channelCache={channelCache}
-                  mySigningAddress={me}
-                />
-              ) : (
-                false
-              )}
-            </>
-          );
-        }}
-      </Web3TorrentContext.Consumer>
+      <>
+        <TorrentInfo torrent={torrent} channelCache={channelCache} mySigningAddress={me} />
+        <br />
+        {showBudget ? (
+          <SiteBudgetTable
+            budgetCache={budgetCache}
+            channelCache={channelCache}
+            mySigningAddress={me}
+          />
+        ) : (
+          false
+        )}
+      </>
+      }
```

Comment:
Ups! You missed one :stuck_out_tongue: 
![image](https://user-images.githubusercontent.com/10502605/79379576-7a4bbe80-7f35-11ea-95ed-3438b697b177.png)

Reply:
Good catch! 37e02c957